src/query/source/source.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
nit: 2021

Reply:
lol i FINALLY installed the correct template in goland, so this should be the last copyright mistake :)